aplus/settings.py

```diff
@@ -526,3 +535,10 @@
         DEBUG_TOOLBAR_CONFIG = {
             'SHOW_TOOLBAR_CALLBACK': 'lib.helpers.show_debug_toolbar',
         }
+
+# Celery
+CELERY_ACCEPT_CONTENT = ['json']
+CELERY_TASK_SERIALIZER = 'json'
+CELERY_RESULT_SERIALIZER = 'json'
+CELERY_BROKER_URL = 'redis://redis:6379'
+CELERY_RESULT_BACKEND = 'redis://redis:6379'
```

Comment:
Is there a reason that these variables are defined in the bottom of the file?

Local_settings.py and environment variables are included on the line 475 and below. Since these variables are defined after that, I think it becomes IMPOSSIBLE to override these CELERY configurations in local settings. Why would you do that?